01PITT_INST-ULS_1_5/js/custom.js

```diff
@@ -267,45 +275,170 @@ angular.module('hathiTrustAvailability', []).constant('hathiTrustBaseUrl', 'http
 });
 
 
-angular.module('getTempAddress', []).controller('prmRequestAfterController',function($scope){
-  //watch for the user to select a pickup location in the hold-request form
-  $scope.$watch(angular.bind($scope.$parent.$ctrl.requestService, function () {
-  if (typeof $scope.$parent.$ctrl.requestService !== 'undefined' && typeof $scope.$parent.$ctrl.requestService._formData !== 'undefined' && typeof $scope.$parent.$ctrl.requestService._formData.pickupLocation !== 'undefined'){
-    //result becomes newVal  
-    return $scope.$parent.$ctrl.requestService._formData.pickupLocation;
-   }
-   //every time the form changes:
-  }), function (newVal) {
-    //fix unecessary gap in instructions area:
-    //makes sure the form is loaded..
-    if (typeof angular.element(document.getElementsByTagName('prm-form-field')) !== 'undefined' && typeof angular.element(document.getElementsByTagName('prm-form-field'))[2] !== 'undefined' && typeof angular.element(document.getElementsByTagName('prm-form-field'))[2].children !== 'undefined'){
-      angular.element(document.getElementsByTagName('prm-form-field'))[2].children[0].children[0].style.marginTop='-25px';
+angular
+  .module('addressSelector', [])
+  .constant('addressServiceBaseUrl', 'https://dev-patron.libraries.pitt.edu/')
```

Comment:
I like how you define the base url in a single place.  I should follow this pattern in the sms branch.  You might be interested in the code Clinton suggested there for detecting the dev or prod site from the primo url.  https://github.com/ulsdevteam/primove-default-view/blob/8b45fee98afe3f6fc39085693bf4e6e7e3f673a3/01PITT_INST-ULS_1_5/js/custom.js#L337